skflow/__init__.py

```diff
@@ -117,6 +117,15 @@ def predict(self, X):
             return pred
         return pred.argmax(axis=1)
 
+    def predict_proba(self, X):
+        pred = self._session.run(self._model.predictions,
```

Comment:
Something like this:
def _predict(self, X):
  return self._session.run(self._model.predictions, feed_dict={ .. })

def predict(self, X):
  pred = self._predict(X)
  if self.n_classes < 2:
    return pred
  return pred.argmax(axis=1)

def predict_proba(self, X):
   return self._predict(X)